tcms/signals.py

```diff
@@ -21,6 +21,7 @@
 """
 from django.dispatch import Signal
 
+from tcms.core.utils.mailto import mailto
```

Comment:
Move the import inside the signal handler body for now.

There's a risk of Django not being loaded/configured properly at startup time b/c I'm not quite sure what is the order in which signal handlers are registered. Maybe this is not a valid concern but let's stick to whatever is working for the time being.